pkg/spec/spec.go

```diff
@@ -188,6 +188,47 @@ func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 		}
 	}
 
+	if strings.HasSuffix(config.Command[0], "init") ||
+		strings.HasSuffix(config.Command[0], "systemd") {
+		options := []string{"rw", "private", "noexec", "nosuid", "nodev"}
+		for _, i := range []string{"/run", "/run/lock", "/sys/fs/cgroup/systemd"} {
+			tmpfsMnt := spec.Mount{
+				Destination: i,
+				Type:        "tmpfs",
+				Source:      "tmpfs",
+				Options:     append(append(options, "tmpcopyup"), "size=65536k"),
```

Comment:
Don't need a double append, append(options, "tmpcopyup", "size=65536k") should work

Reply:
Thought that did not work. Fixed.